.github/workflows/R-CMD-check.yaml

```diff
@@ -51,7 +51,7 @@ jobs:
           # NOTE: this specific branch is currently needed for conga
           python3 -m pip install --user scanpy[leiden]
           cd ../
-          git clone -b rhesus https://github.com/phbradley/conga.git conga
+          git clone https://github.com/phbradley/conga.git conga
```

Comment:
also, I wanted to touch on this. 

This doesn't have anything to do with the PR/diversity metrics beyond being downstream of the CoNGA clones file, but if the CoNGA folk delete the rhesus branch (they just merged it), then our docker/github action runners will break.